src/main/java/frc/utils/ctre/PhoenixProUtils.java

```diff
@@ -1,11 +1,23 @@
 package frc.utils.ctre;
 
+import com.ctre.phoenix6.StatusCode;
 import com.ctre.phoenix6.StatusSignal;
 
+import java.util.function.Supplier;
+
 public class PhoenixProUtils {
 
     public static <T> StatusSignal<T> getRefreshedSignal(boolean refresh, StatusSignal<T> signal) {
         return refresh ? signal.refresh() : signal;
     }
 
+    public static boolean checkAndRetry(Supplier<StatusCode> statusCodeSupplier, int numberOfTries) {
```

Comment:
This function name suggests it has  2 responsibilities, as it has "and" in its name, which is also not very clean and breaks the Single Responsibility Principle. But actually this function does only one thing and that is the retries. Consider a better name for the function.